### DISCUSSION TITLE
Some questions

### DISCUSSION THREAD

**Discussion by Tweak-NL** [JanM321/esphome-lg-controller#77] [Q&A]
At first thank you @JanM321 for creating this. (also sorry i used the issues tab for asking for a PCB)

But i got a PCB now and got a free ESP-S3 from a co-worker but this is the wrong one, this one has 3 more pins on both sides.

![Schermafbeelding 2024-10-09 145303](https://github.com/user-attachments/assets/f81cd568-8bf7-443b-99c6-9f4208927720)

But i was able to find out here that the 3V and GND are on the end and this is the same on this ESP so i changed the GPIO numbers in de base.yml and changed the board and it works!

But can somebody help me to get power usage working ? Im totally a noob in this. And the other question, it is nice the AC wont make that annoying sound anymore after getting a command. But how do know the command is "accepted" by the AC?

Also in the other YML file we need to fill in: temperature_sensor_entity_id: sensor.

I thougt this was something we could choose? Like the build in temp sensor or if you want a other one.

**Comment by JanM321** [comment]
> But can somebody help me to get power usage working ?

This isn't currently supported by this controller and many units don't send the messages for it. It's likely easier (and more reliable) to use a Shelly Plug or HomeWizard Socket.

> I thougt this was something we could choose? Like the build in temp sensor or if you want a other one.

That's the "internal thermistor" switch. If it's on, it will use the sensor in the indoor unit. If it's off, it uses the Home Assistant sensor instead.

**Comment by Tweak-NL** [comment]
Well the SmartThinQ LGE addon (cloud thing in HA) shows the power usage from my AC. But i want to get rid of that cloud addon.

So if i am correct this: 3-5 | XXXX_XXXX | Power Consumption in kWh. Example: 64.91.29 => 64912.9 kWh

Is missing in the lg-controller.h ? Would be nice if someone can implement it even if some AC's dont give the info.

About the internal thermistor yes you can turn it on and off. BUT when you flash the software to the ESP it looks like its required to fill in a alternative sensor for the temp?
